packages/mdc-dialog/index.ts

```diff
@@ -93,7 +93,7 @@ class MDCDialog extends MDCComponent<MDCDialogFoundation> {
     this.buttons_ = [].slice.call(this.root_.querySelectorAll<HTMLElement>(strings.BUTTON_SELECTOR));
     this.defaultButton_ = this.root_.querySelector<HTMLElement>(strings.DEFAULT_BUTTON_SELECTOR);
     this.focusTrapFactory_ = focusTrapFactory;
-    this.initialFocusEl_ = initialFocusEl as HTMLElement;
+    this.initialFocusEl_ = initialFocusEl as HTMLElement | undefined;
```

Comment:
Why do we cast to `HTMLElement` here when we're accepting `Element` above? Is there something we need from `HTMLElement` in particular?

Reply:
I was trying to make things simpler for Closure users (who typically pass an `Element`).

However, this method is called indirectly via the `MDCComponent` constructor (which takes an `Element`), so it actually shouldn't be an issue for them.

I'll change the type of the `initialFocusEl` arg to `HTMLElement`. The TypeScript compiler seems fine with it.

Reply:
Done